Package.swift

```diff
@@ -0,0 +1,38 @@
+// swift-tools-version:5.3
+import PackageDescription
+let package = Package(
+    name: "Cheetah-iOS",
+    platforms: [
+        .iOS(.v13)
+    ],
+    products: [
+        .library(
+            name: "Cheetah",
+            targets: ["Cheetah"]
+        )
+    ],
+    targets: [
+        .binaryTarget(
+            name: "PvCheetah",
+            path: "lib/ios/PvCheetah.xcframework"
+        ),
+        .target(
+            name: "Cheetah",
+            dependencies: ["PvCheetah"],
+            path: ".",
+            exclude: [
+                "binding/ios/CheetahAppTest",
+                "binding/flutter",
+                "binding/react-native",
+                "demo"
+            ],
+            sources: [
+                "binding/ios/Cheetah.swift",
+                "binding/ios/CheetahErrors.swift"
+            ],
+            resources: [
+               .copy("lib/common/cheetah_params.pv")
+            ]
```

Comment:
we don't ship default language model as part of the mobile SDKs (too large). Ensure you're double-checking what resources are mentioned in the podspec file and replicate that.